payments/mobile-pay/other-features.md

```diff
@@ -16,7 +16,819 @@ sidebar:
+
+The `cancellations` resource lists the cancellation transactions on a specific
+payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/mobilepay/payments/{{ page.paymentId }}/cancellations HTTP/1.1
+Host: api.payex.com
+Authorization: Bearer <MerchantToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/mobilepay/payments/{{ page.paymentId }}",
+    "cancellations": {
+        "id": "/psp/mobilepay/payments/{{ page.paymentId }}/cancellations",
+        "cancellationList": [
+            {
+                "id": "/psp/mobilepay/payments/{{ page.paymentId }}/cancellations/{{ page.transactionId }}",
+                "transaction": {
+                    "id": "/psp/mobilepay/payments/{{ page.paymentId }}/transactions/{{ page.transactionId }}",
+                    "created": "2018-09-11T12:19:38.1247314Z",
+                    "updated": "2018-09-11T12:19:38.3059149Z",
+                    "type": "Cancellation",
+                    "state": "Completed",
+                    "number": 75100000127,
+                    "amount": 500,
+                    "vatAmount": 0,
+                    "description": "Test Cancellation",
+                    "payeeReference": "ABC123",
+                    "isOperational": false,
+                    "operations": []
+                }
+            }
+        ]
+    }
+}
+```
+
+{:.table .table-striped}
+| Property                     | Type     | Description                                                                         |
+| :--------------------------- | :------- | :---------------------------------------------------------------------------------- |
+| `payment`                    | `string` | The relative URI of the payment this list of cancellation transactions belong to.   |
+| `cancellations`              | `object` | The current `cancellations` resource.                                               |
+| └➔&nbsp;`id`                 | `string` | The relative URI of the current `cancellations` resource.                           |
+| └➔&nbsp;`cancellationList`   | `array`  | The array of the cancellation transaction objects.                                  |
+| └➔&nbsp;`cancellationList[]` | `object` | The object representation of the cancellation transaction resource described below. |
+
+### Create cancellation transaction
+
+Perform the `create-cancel` operation to cancel a previously created payment.
+You can only cancel a payment - or part of payment - not yet captured.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/mobilepay/payments/{{ page.paymentId }}/cancellations HTTP/1.1
+Host: api.payex.com
+Authorization: Bearer <MerchantToken>
+Content-Type: application/json
+
+{
+    "transaction": {
+        "description": "Test Cancellation",
+        "payeeReference": "ABC123"
+    }
+}
+```
+
+{:.table .table-striped}
+| Property| Type| Required| Description
```

Comment:
The `Required` column should be the first column of the table and we should use the unicode checkmark and not `Y` as a marker for required properties.